inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -387,8 +402,9 @@ def error_workflow(message):
     @wraps(error_workflow)
     def _error_workflow(obj, eng):
         obj.log.error(message)
-        obj.extra_data['_error_message'] = message
+        obj.extra_data['_error_msg'] = message
```

Comment:
@jacquerie I will remove line 405 and 406, here and also in `error_workflow` action because it is already done as I raise a `WorkflowsError`. Check [this] (https://github.com/inveniosoftware-contrib/invenio-workflows/blob/1b774eb6d61ec75b4b83d06cc59b7e3f919c2db5/invenio_workflows/engine.py#L342). Please correct me if I am wrong.